bpf/grantedv2.h

```diff
@@ -0,0 +1,172 @@
+/*
+ * Gatekeeper - DoS protection system.
+ * Copyright (C) 2016 Digirati LTDA.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#include <rte_common.h>
+
+#include "gatekeeper_flow_bpf.h"
+#include "bpf_mbuf.h"
+
+struct grantedv2_params {
+	/*
+	 * Primary rate limit: kilobyte/second.
```

Comment:
A typo: ``kilobytes/second``. The same applies to the comment below.

Reply:
Given it is a unit, it's not a typo. A unit can be written using their names in singular forms.